mne/decoding/time_gen.py

```diff
@@ -108,36 +123,57 @@ class GeneralizationAcrossTime(object):
                 these predictions into a single estimate per sample.
 
         Default: 'cross-validation'
+    scorer : object | None
+        scikit-learn Scorer instance. If None, set to accuracy_score.
+        Defaults to None.
     n_jobs : int
         Number of jobs to run in parallel. Defaults to 1.
 
     Attributes
     ----------
-    scorer_ : object
-        scikit-learn Scorer instance. Default: accuracy_score
+    picks_ : array-like of int
+        Channels to be included.
+    ch_names : list, shape (n_channels,)
+        Names of the channels used for training.
     y_train_ : list | np.ndarray, shape (n_samples,)
         The categories used for training.
+    train_times_ : dict
+        A dictionary that configures the training times:
+
+            ``slices`` : np.ndarray, shape (n_clfs,)
+                Array of time slices (in indices) used for each classifier.
+                If not given, computed from 'start', 'stop', 'length', 'step'.
+            ``times`` : np.ndarray, shape (n_clfs,)
+                The training times (in seconds).
+
+    test_times_ : dict
+        A dictionary that configures the testing times for each training time:
+
+            ``slices`` : np.ndarray, shape (n_clfs, n_testing_times)
+                Array of time slices (in indices) used for each classifier.
+            ``times`` : np.ndarray, shape (n_clfs, n_testing_times)
+                The testing times (in seconds) for each training time.
+
     estimators_ : list of list of sklearn.base.BaseEstimator subclasses.
         The estimators for each time point and each fold.
+    cv_ : CrossValidation object
+        The actual CrossValidation input depending on y.
```

Comment:
point taken, but I think it's better to be stricter for now. Otherwise we'll have to detail how it's supposed to be constructed to be meaningful (i.e. what happens if some samples are not given, or are duplicated).